inspirehep/modules/theme/templates/inspirehep_theme/accounts/settings/profile.html

```diff
@@ -18,60 +18,76 @@
-  <div class="alert alert-warning custom-alert">
-  {{ _("You have not yet verified your email address.") }} {{ verification_form.send_verification_email(class_="btn btn-default btn-xs") }}
-  </div>
-<input type="hidden" name="submit" value="verification" />
-</form>
-{% endif %}
-<div class="row"><div class="col-md-12"><p></p></div></div>
-<div class="row" id="top-row">
-  <div class="col-lg-2" id="user-icon">
-    <img id="profile-picture" class="img-round" src="{{ current_user.email | gravatar(size=90, default='identicon') }}" data-toggle='tooltip' data-placement='left' title='Your profile picture can be set at gravatar.com'>
-  </div>
-  <div class="col-lg-10 profile-summary">
-    {%- if current_user.given_names or current_user.family_name %}
-      <h4 class="media-heading">{{ current_user.given_names ~ " " ~ current_user.family_name }}</h4>
-    {%- else %}
-      <h4 class="media-heading">{{ current_user.nickname }}</h4>
-    {%- endif %}
-    <p class="text-muted">{{ current_user.email }}</p>
+{% block body %}
+<div class="row">
+  <div class="col-md-3">
+    <div class="panel-group">
+    <div class="panel panel-default">
+      <div class="panel-heading"><strong>Settings</strong></div>
+      <ul class="list-group">
+        {%- for item in current_menu.submenu('settings').children if item.visible %}
+        <li class="list-group-item{% if item.active %} active custom-active{% endif %}"><a class="link-display" href="{{ item.url }}">{{ item.text|safe }}</a></li>
+        {%- endfor %}
+      </ul>
+    </div>
+  </div>    
   </div>
-</div>
-<div class="row" id="settings-form">
-  <div class="col-sm-11 col-md-11">
-    <form method="POST" role="form">
-      {%- for field in profile_form -%}
-        {{ render_field(field) }}
-      {%- endfor -%}
-      <div class="form-actions">
-        <button type="submit" name="submit" value="profile" class="btn btn-success"><i class="fa fa-check"></i> {{ _("Update Profile") }}</button>
+  <div class="col-md-9">
+  <div class="panel panel-default custom-panel">
+    <div class="panel-heading"><i class="fa fa-user fa-fw"></i> <strong>Profile</strong></div>
+      <div class="panel-body">
+      {% if current_user.note == "2" and verification_form %}
+      <form method="POST" role="form">
+      {{ verification_form.csrf_token }}
+        <div class="alert alert-warning custom-alert">
+        {{ _("You have not yet verified your email address.") }} {{ verification_form.send_verification_email(class_="btn btn-default btn-xs") }}
+        </div>
+      <input type="hidden" name="submit" value="verification" />
+      </form>
+      {% endif %}
+      <div class="row"><div class="col-md-12"><p></p></div></div>
+      <div class="row" id="top-row">
+        <div class="col-lg-2" id="user-icon">
+          <img id="profile-picture" class="img-round" src="{{ current_user.email }}" data-toggle='tooltip' data-placement='left' title='Your profile picture can be set at gravatar.com'>
```

Comment:
@jmartinm  here there was the line src="{{ current_user.email | gravatar(...) }}" but I get an error that gravatar is not a filter, although I installed invenio_ext. Ideas?